### ISSUE TITLE
ds -> data-system in "cozy-monitor status"

### ISSUE THREAD

**Issue by nledez** [cozy/cozy-monitor#82]
Hello,

when I made a "cozy-monitor status" I would like to have "data-system" instead "ds":
```
root@cozy-test-nlz:~# cozy-monitor status
postfix: up
couch: up
controller: up
data-system: up
home: up
proxy: up
index: up
```

**Comment by poupotte** [comment]
It's fixed with 3d20e13

**Comment by nledez** [comment]
Thanks! ❤️🌹
